db/generators/generateData.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const generateInstructors = require('./instructorsGenerator');
+const generateOfferedBys = require('./offeredBysGenerator');
+const generateTestimonials = require('./testimonialsGenerator');
+
+const totalRecords = 100;
+const instructors = generateInstructors(totalRecords);
+const offeredBys = generateOfferedBys(totalRecords);
+const testimonials = generateTestimonials(totalRecords);
+
+fs.writeFileSync('./db/data/instructors.json', JSON.stringify(instructors, null, '\t'));
```

Comment:
writing is an expensive operation when it comes time to benchmark (something to keep in mind)

Reply:
Thanks! I have this commented out for the next PR. I just read writing to file is very taxing. By omitting this I was able to increase the data generation from 100,000 to 1,000,000 records without maxing out the heap.